Loadable.svelte

```diff
@@ -127,15 +130,17 @@
     state = STATES.SUCCESS
     component = LOADED.get(loader)
   } else {
-    onMount(async () => {
-      await load()
-      dispatch('load')
-      if (unloader) {
-        return () => {
-          LOADED.delete(loader)
-          if (typeof unloader === 'function') {
-            unloader()
-          }
+    onMount(() => {
+      Promise.resolve(load()).then(() => {
+        if (mounted) {
+          dispatch('load')
+        }
+      })
```

Comment:
We can set `mounted` to `true` on the... `onMount` method 😁 

```suggestion
      mounted = true;
      load().then(() => {
        if (mounted) {
          dispatch('load')
        }
      })
```

Reply:
good call - fixed